packages/react-native-reanimated/src/mutables.ts

```diff
@@ -1,11 +1,14 @@
 'use strict';
 import { shouldBeUseWeb } from './PlatformChecker';
 import type { Mutable } from './commonTypes';
+
 import { shareableMappingCache } from './shareableMappingCache';
 import { makeShareableCloneRecursive } from './shareables';
 import { executeOnUIRuntimeSync, runOnUI } from './threads';
 import { valueSetter } from './valueSetter';
 
+const logger = global.__reanimatedLogger;
```

Comment:
All usages of `global.__reanimatedLogger` that happened before this file was evaluated will be broken.